src/server/modules/api/external-v1/index.ts

```diff
@@ -13,14 +13,19 @@ type ShortUrlProperty = {
   shortUrl: string
 }
 
+type OptionalEmailProperty = {
+  email?: string
+}
+
 type ShortUrlOperationProperty = UserIdProperty & ShortUrlProperty
 
 type OptionalStateProperty = {
   state?: 'ACTIVE' | 'INACTIVE'
 }
 
 export type UrlCreationRequest = ShortUrlOperationProperty &
-  OptionalLongUrlProperty
+  OptionalLongUrlProperty &
+  OptionalEmailProperty
```

Comment:
oh yes, since we're no longer using the same module! removed in [b4c75ec](https://github.com/opengovsg/GoGovSG/pull/2213/commits/b4c75ecebe7d6e0f5466dfd7e4c8fd0d60db90a7) thanks!